sys/usb/usbus/usbus_control.c

```diff
@@ -272,9 +272,7 @@ static void _recv_setup(usbus_t *usbus, usbus_control_handler_t *handler)
     }
     if (res < 0) {
         /* Signal stall to indicate unsupported (USB 2.0 spec 9.6.2 */
```

Comment:
```suggestion
        /* Signal stall to indicate unsupported (USB 2.0 spec 9.6.2) */
```
Unrelated but while we're here...

Reply:
sure, fixed